SwatDB/SwatDBDataObject.php

```diff
@@ -212,6 +221,13 @@ public function __set($key, $value)
 			return;
 		}
 
+		$property_list = $this->getProtectedPropertyList();
+		if (array_key_exists($key, $property_list)) {
+			$set = $property_list[$key]['set'];
```

Comment:
Need to check if setter exists. If it doesn't exist, should we just set the protected property (default setter)?